### PR TITLE
chore: bump model-viewer and transpile via babel

### LINKED ISSUE #2852: Check on @google/model-viewer@^1.11.1
Check on new recent latest google/model-viewer 

### Backlog
Seems to skip @google/model-viewer for now
- [x] add to ignore packages into renovate.json 
- [x] make an issue about it to add why it's there if someone wants to tackle that in parallel
- [x] update minor packages :)
image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/2811#issuecomment-1094902743_

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2852
- [x] bump model-viewer and transpile via babel to make sure future versions can be loaded correctly

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -266,6 +266,24 @@ export default defineNuxtConfig({
...
+              {
+                test: './vendor/large.min.js',
+                compact: true,
+              },
+            ],
+          },
+        ],
+      ]
+    },
+  },
```
Comment (roiLeo) [inline nuxt.config.js:285]: What is the purpose of this? only put model-viewer in transpile doesn't work?

Comment (kkukelka) [reply nuxt.config.js]: it is there to silence a rather unimportant warning by babel https://github.com/babel/babel/discussions/13676

i just pushed a much more simple config and added a comment regarding its purpose

**Review by roiLeo — APPROVED** [review]
lgtm

**Comment by roiLeo** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 174 USD/KSM ~ 0.287 $KSM
🧗 [EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW](https://kusama.subscan.io/account/EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)
🔗 [0x9bb7e9f6c7f4a0cb270528cdf66cc8395451b21c890de0800d15089d388d6fcd](https://kusama.subscan.io/extrinsic/0x9bb7e9f6c7f4a0cb270528cdf66cc8395451b21c890de0800d15089d388d6fcd)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**